CMake/QtDCMConfig.cmake.in

```diff
@@ -2,51 +2,36 @@
 # This file is to be included from outside projects which uses QtM3d
 
 #path to the source of QTDCM
-SET(QTDCM_SOURCE_DIR "@QTDCM_SOURCE_DIR@/Code")
+set(QTDCM_SOURCE_DIR "@QtDcm_SOURCE_DIR@/Code")
 
 #path to cmake files of Vistal
-SET(QTDCM_CMAKE_DIR "@QTDCM_CMAKE_DIR@")
+set(QTDCM_CMAKE_DIR "@QtDcm_CMAKE_DIR@")
 
 #path to binary dir of QTDCM
-SET(QTDCM_BINARY_DIR "@QTDCM_BIN_DIR@")
+set(QTDCM_BINARY_DIR "@QtDcm_BINARY_DIR@")
 
 # all include dirs pour QTDCM
-SET(QTDCM_INCLUDE_DIRS  "@QTDCM_INCLUDE_DIRS@")
+set(QTDCM_INCLUDE_DIRS  "@QtDcm_INCLUDE_DIRS@")
 
 # all library dirs pour QTDCM
-SET(QTDCM_LIBRARY_DIRS  "@QTDCM_LIBRARY_DIRS@")
+set(QTDCM_LIBRARY_DIRS  "@QtDcm_LIBRARY_DIRS@")
 
```

Comment:
The `QtDcm_LIBRARY_DIRS` is empty, it is not defined somwhere else in the project. the `set(QTDCM_SOURCE_DIR "@QtDcm_SOURCE_DIR@/Code")` above works because CMake automatically internally define the `${${PROJECT_NAME}_BINARY_DIR}`  and `${${SOURCE_NAME}_BINARY_DIR}`

I don't know how you want to deal with all of it, however i made a quick fix, see https://github.com/aabadie/qtdcm/pull/4